test/putTransferSpec.js

```diff
@@ -1016,13 +1016,13 @@ describe('PUT /transfers/:id', function () {
       .post('/notifications', (body) => {
         const idParts = body.id.split('/')
         const notificationId = idParts[idParts.length - 1]
-        expect(body).to.deep.equal({
+        expect(_.omit(body, 'signature')).to.deep.equal({
           event: 'transfer.update',
           id: subscription.id + '/notifications/' + notificationId,
           subscription: subscription.id,
           resource: transferResult
         })
-        // TODO validate this against notification schema
+        expect(validator.validateNotification.bind(validator.validateNotification, {body: body})).to.not.throw(Error)
```

Comment:
If my understanding of `_.omit` is correct (it doesn't mutate the source object), it will omit the signature when checking the deep equality of the two objects (because the signature changes each time), but not when actually validating against the schema.